rust/automerge/src/op_set2/change.rs

```diff
@@ -41,16 +41,35 @@ pub(crate) fn build_change<T, G>(
     graph: &G,
     actors: &[ActorId],
 ) -> Change<'static, Verified>
+where
+    T: AsChangeOp,
+    G: GetHash,
+{
```

Comment:
Might be worth a comment that we are doing this so we can reuse the memory for the actor map as it made a big difference to performance. Otherwise 6 months down the road we'll "simplify" this away.